community/assessments/projects/kubescape/self-assessment.md

```diff
@@ -0,0 +1,337 @@
+
+#### Node agent
+
+*Node agent* is completely event-driven and it does not expose a transactional API.
+
+It responds to:
+
+* Kubernetes API events: Pods that are created, removed on a node
+* Container runtime events: Starting monitoring when a container starts and stopping it when it is stopped.
+* *Node agent* lifecycle events: When the component starts, an automatic configuration scan of the node is triggered
+* *OperatorCommand* objects: such objects convey command to the *Node agent* to delete pods/processes as a response to a detection
+* *RuntimeRuleAlertBinding* objects: these objects define which detection rules should be applied to which Pods
+
+##### Synchronizer
+
+This component is also entirely event-driven, and two actors trigger its actions:
+
+* Kubernetes API: *Synchronizer* responds to changes in the Kubernetes API objects - this triggers an action to send these objects to the connected Synchronizer server
+* Synchornizer server: Synchornizer server can trigger the *Synchronizer* component to store objects in Kubernetes API (this is limited by the *ClusterRole* the *Synchronizer* is assigned with)
+
+### Goals
+
+* The Kubescape Operator automates scanning and monitoring of Kubernetes clusters
+  * provides up-to-date security posture data
+  * creates proposals on how to improve the security posture of the workload and the cluster
+  * provides security observability data
+  * alerts on potential breaches to workloads in the cluster
+* Kubescape CLI scans pre-deployment and post-deployment configurations and images and provides actionable results
+
+#### Security Goals
+
+* All access to the data produced by Kubescape Operator should be only available to those subjects who have the proper RBAC authorizations
+* Kubescape should only share information to מon-Kubernetes endpoints in cases when the user explicitly defined it
+* Kubescape should only have access to objects it needs to complete its job
+* Triggers should only come from authenticated and authorized sources
+
+### Non-goals
+
+* Kubescape is not meant to enforce access control beyond what the Kubernetes API already provides
+
+## Self-assessment use
+
+This self-assessment is not intended to provide a security audit of Kubescape or function as an independent assessment or attestation of Kubescape's security health.
+
+This document provides Kubescape users with an initial understanding of Kubescape's security, where to find existing security documentation, Kubescape's plans for security, and a general overview of Kubescape's security practices, both for the development of Kubescape and its operational security.
+
+This document provides the CNCF TAG-Security with an initial understanding of Kubescape to assist in a joint assessment necessary for projects under incubation. Taken together, this document and the joint assessment serve as a cornerstone for if and when Kubescape is preparing for a security audit.
```

Comment:
Joint assessments are not required for incubation.  This piece of boilerplate is for _incubating_ projects to move to graduation.

```suggestion
```